src/docker/ContextManager.ts

```diff
@@ -122,11 +120,15 @@ export class DockerContextManager implements ContextManager, Disposable {
                 // But that probably won't be true in the future, so define both as separate concepts now
                 await this.setVsCodeContext('vscode-docker:aciContext', true);
                 await this.setVsCodeContext('vscode-docker:newSdkContext', true);
-                ext.dockerClient = new DockerServeClient(currentContext);
+
+                const dsc = await import('./DockerServeClient/DockerServeClient');

```

Comment:
Would it make sense to cache the result of this import (and the dockerode one)?

Reply:
I think the module system does internal caching... ? 🤔

Reply:
Yeah, it does it internally. I set some logpoints before and after this; the first time it was about 200ms (unpacked), every subsequent time it was 0ms. So no need to cache ourselves.